vuepress/docs/v6.3.2/tutorials/samples/hub.md

```diff
@@ -51,33 +51,33 @@ TODO: screenshot of the 6.3.2 AppBuilder with both bundles available but not ins
 :::
 
-5. The Hub can be accessed from the App Builder by navigating to `Pages → Management`, finding `xxxx` in the page tree, and clicking `View Published Page` from its actions.
+5. The Hub can be accessed from the App Builder by navigating to `Pages → Management`, finding `Hub Test` **change** in the page tree, and clicking `View Published Page` from its actions.
 
 ## Configuration
 
 ::: tip
-(New with Entando 7.0.0) To point the Entando App Builder in another Entando Application to any Hub instance, configure the App Builder with the URL for the Hub API: http://YOUR-BASE-URL/entando-hub-api/appbuilder/api.  For example, the public Entando Hub has this API: <http://hubdev.okd-entando.org/entando-hub-api/appbuilder/api>. **change**
+(New with Entando 7.0.0) To point the Entando App Builder in another Entando Application to any Hub instance, configure the App Builder using the endpoint `BASEURL/entando-hub-api/appbuilder/api`, where the BASEURL is the URL for the Entando Application.
 :::
 
 ## Using the Hub
 
 ### Concepts
 
-These are the key entities in the Entando Hub.
+These are the key entities in the Entando Hub:
 
 - `Bundle Group`: A Bundle Group is a group of one or more Entando Bundles. 
 - `Bundle Group Version`: A Bundle Group can have one or more versions, each with a particular status.
-- `Bundle`: A Bundle is the deployment unit within an Entando Application. A Bundle can contain one or multiple components such as micro frontends, microservices, or any of the [component types](../TODO) available in Entando. 
+- `Bundle`: A Bundle is the deployment unit within an Entando Application. A Bundle can contain one or multiple components such as micro frontends, microservices, or any of the [component types](../../docs/ecr/ecr-bundle-details.html#overview) available in Entando. 
 - `Category`: Each Bundle Group belongs to a specific category. The initial possible categories are Solution Template, Packaged Business Capability (PBC), or Component Collection. An admin of an Entando Hub can refine the available categories as desired.
 - `Organization`: Bundle Groups belong to a single organization. Authors and managers can only update Bundle Groups for their own organization.  
 - `User`: User identity is managed within Keycloak, where users are granted roles within the Hub. Users must be assigned to a specific organization.
 
 Notes:
-- A private repository can be used for a Bundle, but this requires [an additional Kubernetes secret](../TODO) before deployment via the App Builder.
+- A private repository can be used for a Bundle, but this requires [an additional Kubernetes secret](../../docs/reference/identity-management.html#logging-into-your-keycloak-instance) before deployment via the App Builder.
 
 ### Roles
 
-Three roles are used to provide access to the Hub features.
+Three roles are used to provide access to the Hub features:
```

Comment:
this is minor but is access to the Hub different than access to the Hub features? Because suddenly instead of Hub, it says Hub features for access, it made me pause.

Reply:
yes - different roles can access different features, like changing fields or deleting versions